Jason/Coin_Toss.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Python Fundamentals Assignment
+
+Autho: Jason Lee
+
+"""
+import random
+
+head_count = 0
+tail_count = 0
+
+for num in range(1,5001):
+    if(random.random()>0.5):
+        head_count += 1
+        print "Attemp #" + str(num) + ": Throwing a coin ... It's a head! ... Got " \
```

Comment:
This print state is the exact same whether the coin is head or tails side up. Seems like the should be slightly differennt.